fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubberTest.java

```diff
@@ -313,8 +313,8 @@ void testScrubberInvalidIndexState() {
         openSimpleMetaData(hook2);
         buildIndexClean(nonValueIndex);
         try (OnlineIndexScrubber indexScrubber = newScrubberBuilder(nonValueIndex).build()) {
```

Comment:
Probably yes. Should it be part of a specific maintainers test? Maybe we can start with Lucene's.

Reply:
I feel like starting with the existing indexes would be more valuable, and easier to reason about.
Then again, adding Lucene support may cause us to have to alter the api here, so it might be better to defer those tests until we have a scrubber for another index type.